autogpt_platform/frontend/src/lib/supabase/hooks/useSupabase.ts

```diff
@@ -34,9 +34,7 @@ export function useSupabase() {
 
     broadcastLogout();
 
-    const { error } = await supabase.auth.signOut({
-      scope: "global",
-    });
```

Comment:
Taking opportunity to fix this too. I believe we only want global sign out on reset password 👀